File: CHPT/source/tags/V0.4.1/client_app/Model/tb_workhours.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Model
{
    //工时档案
    public class tb_workhours
    {

        /// <summary>
        /// 工时档案ID
        /// </summary>		
        private string _whours_id;
        public string whours_id
        {
            get { return _whours_id; }
            set { _whours_id = value; }
        }
        /// <summary>
        /// 数据来源，关联字典码表
        /// </summary>		
        private string _data_sources;
        public string data_sources
        {
            get { return _data_sources; }
            set { _data_sources = value; }
        }
        /// <summary>
        /// 维修项目类别，关联字典码表
        /// </summary>		
        private string _repair_type;
        public string repair_type
        {
            get { return _repair_type; }
            set { _repair_type = value; }
        }
        /// <summary>
        /// 项目编号
        /// </summary>		
        private string _project_num;
        public string project_num
        {
            get { return _project_num; }
            set { _project_num = value; }
        }
        /// <summary>
        /// 项目名称
        /// </summary>		
        private string _project_name;
        public string project_name
        {
            get { return _project_name; }
            set { _project_name = value; }
        }
        /// <summary>
        /// 项目备注
        /// </summary>		
        private string _project_remark;
        public string project_remark
        {
            get { return _project_remark; }
            set { _project_remark = value; }
        }
        /// <summary>
        /// 工时类型，1工时，2定额
        /// </summary>		
        private string _whours_type;
        public string whours_type
        {
            get { return _whours_type; }
            set { _whours_type = value; }
        }
        /// <summary>
        /// A类工时数
        /// </summary>		
        private decimal _whours_num_a;
        public decimal whours_num_a
        {
            get { return _whours_num_a; }
            set { _whours_num_a = value; }
        }
        /// <summary>
        /// B类工时数
        /// </summary>		
        private decimal _whours_num_b;
        public decimal whours_num_b
        {
            get { return _whours_num_b; }
            set { _whours_num_b = value; }
        }
        /// <summary>
        /// C类工时数
        /// </summary>		
        private decimal _whours_num_c;
        public decimal whours_num_c
        {
            get { return _whours_num_c; }
            set { _whours_num_c = value; }
        }
        /// <summary>
        /// 定额单价
        /// </summary>		
        private decimal _quota_price;
        public decimal quota_price
        {
            get { return _quota_price; }
            set { _quota_price = value; }
        }
        /// <summary>
        /// 状态
        /// </summary>		
        private int _status;
        public int status
        {
            get { return _status; }
            set { _status = value; }
        }
        /// <summary>
        /// 最后编辑时间
        /// </summary>		
        private long _update_time;
        public long update_time
        {
            get { return _update_time; }
            set { _update_time = value; }
        }
        /// <summary>
        /// 创建人，关联人员表
        /// </summary>		
        private string _create_by;
        public string create_by
        {
            get { return _create_by; }
            set { _create_by = value; }
        }
        /// <summary>
        /// 创建时间
        /// </summary>		
        private long _create_time;
        public long create_time
        {
            get { return _create_time; }
            set { _create_time = value; }
        }
        /// <summary>
        /// 最后编辑人，关联人员表
        /// </summary>		
        private string _update_by;
        public string update_by
        {
            get { return _update_by; }
            set { _update_by = value; }
        }
        /// <summary>
        /// 删除标记，0为删除，1未删除  默认1
        /// </summary>		
        private string _enable_flag;
        public string enable_flag
        {
            get { return _enable_flag; }
            set { _enable_flag = value; }
        }

    }
}

File: CHPT/source/trunk/supp_app/SspDispatchService/src/com/ctfo/storage/model/basedata/TbWorkhours.java
package com.ctfo.storage.model.basedata;

import java.math.BigDecimal;

public class TbWorkhours {
	/**
	 * tb_workhours.whours_id 工时档案ID
	 */
	private String whoursId;

	/**
	 * tb_workhours.data_source 数据来源
	 */
	private String dataSource;

	/**
	 * tb_workhours.repair_type 维修项目类别，关联字典码表
	 */
	private String repairType;

	/**
	 * tb_workhours.project_num 项目编号
	 */
	private String projectNum;

	/**
	 * tb_workhours.project_name 项目名称
	 */
	private String projectName;

	/**
	 * tb_workhours.project_remark 项目备注
	 */
	private String projectRemark;

	/**
	 * tb_workhours.whours_change 工时调整
	 */
	private String whoursChange;

	/**
	 * tb_workhours.whours_type 工时类型，1工时，2定额
	 */
	private String whoursType;

	/**
	 * tb_workhours.whours_num_a A类工时数
	 */
	private BigDecimal whoursNumA;

	/**
	 * tb_workhours.whours_num_b B类工时数
	 */
	private BigDecimal whoursNumB;

	/**
	 * tb_workhours.whours_num_c C类工时数
	 */
	private BigDecimal whoursNumC;

	/**
	 * tb_workhours.quota_price 定额单价
	 */
	private BigDecimal quotaPrice;

	/**
	 * tb_workhours.status 状态 0，停用 1，启用
	 */
	private String status;

	/**
	 * tb_workhours.enable_flag 删除标记，0为删除，1未删除 默认1
	 */
	private String enableFlag;

	/**
	 * tb_workhours.update_time 最后编辑时间
	 */
	private Long updateTime;

	/**
	 * tb_workhours.create_by 创建人，关联人员表
	 */
	private String createBy;

	/**
	 * tb_workhours.create_time 创建时间
	 */
	private Long createTime;

	/**
	 * tb_workhours.update_by 最后编辑人，关联人员表
	 */
	private String updateBy;

	/** 服务站id，云平台用 */
	private String serStationId;

	/** 帐套id，云平台用 */
	private String setBookId;

	public String getSerStationId() {
		return serStationId;
	}

	public void setSerStationId(String serStationId) {
		this.serStationId = serStationId;
	}

	public String getSetBookId() {
		return setBookId;
	}

	public void setSetBookId(String setBookId) {
		this.setBookId = setBookId;
	}

	public String getWhoursId() {
		return whoursId;
	}

	public void setWhoursId(String whoursId) {
		this.whoursId = whoursId;
	}

	public String getDataSource() {
		return dataSource;
	}

	public void setDataSource(String dataSource) {
		this.dataSource = dataSource;
	}

	public String getRepairType() {
		return repairType;
	}

	public void setRepairType(String repairType) {
		this.repairType = repairType;
	}

	public String getProjectNum() {
		return projectNum;
	}

	public void setProjectNum(String projectNum) {
		this.projectNum = projectNum;
	}

	public String getProjectName() {
		return projectName;
	}

	public void setProjectName(String projectName) {
		this.projectName = projectName;
	}

	public String getProjectRemark() {
		return projectRemark;
	}

	public void setProjectRemark(String projectRemark) {
		this.projectRemark = projectRemark;
	}

	public String getWhoursChange() {
		return whoursChange;
	}

	public void setWhoursChange(String whoursChange) {
		this.whoursChange = whoursChange;
	}

	public String getWhoursType() {
		return whoursType;
	}

	public void setWhoursType(String whoursType) {
		this.whoursType = whoursType;
	}

	public BigDecimal getWhoursNumA() {
		return whoursNumA;
	}

	public void setWhoursNumA(BigDecimal whoursNumA) {
		this.whoursNumA = whoursNumA;
	}

	public BigDecimal getWhoursNumB() {
		return whoursNumB;
	}

	public void setWhoursNumB(BigDecimal whoursNumB) {
		this.whoursNumB = whoursNumB;
	}

	public BigDecimal getWhoursNumC() {
		return whoursNumC;
	}

	public void setWhoursNumC(BigDecimal whoursNumC) {
		this.whoursNumC = whoursNumC;
	}

	public BigDecimal getQuotaPrice() {
		return quotaPrice;
	}

	public void setQuotaPrice(BigDecimal quotaPrice) {
		this.quotaPrice = quotaPrice;
	}

	public String getStatus() {
		return status;
	}

	public void setStatus(String status) {
		this.status = status;
	}

	public String getEnableFlag() {
		return enableFlag;
	}

	public void setEnableFlag(String enableFlag) {
		this.enableFlag = enableFlag;
	}

	public Long getUpdateTime() {
		return updateTime;
	}

	public void setUpdateTime(Long updateTime) {
		this.updateTime = updateTime;
	}

	public String getCreateBy() {
		return createBy;
	}

	public void setCreateBy(String createBy) {
		this.createBy = createBy;
	}

	public Long getCreateTime() {
		return createTime;
	}

	public void setCreateTime(Long createTime) {
		this.createTime = createTime;
	}

	public String getUpdateBy() {
		return updateBy;
	}

	public void setUpdateBy(String updateBy) {
		this.updateBy = updateBy;
	}
}